### DISCUSSION TITLE
Need a REFRESH button to reload json file from a synced folder

### DISCUSSION THREAD

**Discussion by DeeDeeMcCoy** [massCodeIO/massCode#402] [Ideas]
My synced folder works well, and the json file in it is synced well, too. But every time I make some changes, other opened massCode App won't load this json file automatically. So, I have to restart the App to load my latest version json file. Could you add a function to reload the updated json file manually or automatically inside the App? I'll appreciate it.

**Comment by rheber** [comment]
Seconding this, though I think automatic reloading would be ideal so my colleagues and I who share a snippets file don't accidentally clobber each other's changes.

**Comment by elPixelador** [comment]
I had previously suggested official support for cloud drives but thinking about it this would be a much more straight forward to implement.

**Comment by antonreshetov** [comment]
Added a "Reload Storage" button https://github.com/massCodeIO/massCode/pull/408

![image](https://github.com/massCodeIO/massCode/assets/17480987/083a4aab-ca9b-4c43-92e7-73851814c594)
